src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -6,3 +6,96 @@
 from azure.cli.help_files import helps #pylint: disable=unused-import
 
 #pylint: disable=line-too-long
+
+helps['network dns record-set create'] = """
```

Comment:
I filed an issue for this area.  I'd like to leave this help in place since the SDK help is confusing and the issue suggests rolling our help back if we can get the SDK to update their text

---
In reply to: [75752065](https://github.com/Azure/azure-cli/pull/718#discussion_r75752065) [](ancestors = 75752065,75560645,75556265)